### PR TITLE
remove old route actions

### PR DESCRIPTION
**Pull request by chrisnojima**
I was trying to track down some routing errors and wanted to update the logging. Along the way I cleaned up the old route actions. There's a ticket to rethink them but short term  just collapsed the ones we treat as the same and removed params we ignore

- [x] `navigateTo` / `switchTo` => `navigateAppend`
- [x] removed parentPath
- [x] `switchRouteDef` => `switchLoggedIn`

### CODE REVIEW

**Comment by chrisnojima** [comment]
@keybase/react-hackers

**Comment by chrisnojima** [comment]
@keybase/react-hackers bump

File: shared/actions/json/route-tree.json [hunk 1]
```diff
@@ -29,6 +14,10 @@
       "_description": "ONLY used by the new nav. Navigates up to this route if it already exists, noops otherwise.",
       "routeName": "string"
     },
+    "switchLoggedIn": {
+      "_description": "ONLY used by the new nav. Switch login stacks",
```
Comment (jakob223) [inline shared/actions/json/route-tree.json]: is it time to delete the "only used by thee new nav" comments here?

Comment (chrisnojima) [reply shared/actions/json/route-tree.json]: we'll be doing a larger refactor of this soon

**Pull request merged**